src/plugins/identity/alias.test.js

```diff
@@ -77,6 +78,16 @@ describe("src/plugins/identity/alias", () => {
         const b = resolveAlias("discourse/@login", url);
         expect(a).toEqual(b);
       });
+      it("a sourcecred identity", () => {
+        const actual = resolveAlias("sourcecred/example", null);
+        const expected = identityAddress("example");
+        expect(actual).toEqual(expected);
+      });
+      it("a discourse login with prefixed @", () => {
```

Comment:
Nit: "discourse login" should be "sourcecred identity"

Reply:
Done.